### PR TITLE
dotenv-linter 2.2.1 (new formula)

### PR DESCRIPTION
**Pull request by vladimyr**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Related to https://github.com/dotenv-linter/dotenv-linter/issues/114

### CODE REVIEW

File: Formula/dotenv-linter.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      _FOO=BAR
+    EOS
+    expected = <<~EOS
+      .env:2 DuplicatedKey: The FOO key is duplicated
+      .env:3 UnorderedKey: The BAR key should go before the FOO key
+      .env.test:1 LeadingCharacter: Invalid leading character detected
+
+      Found 3 problems
+    EOS
+    assert_equal expected, shell_output("#{bin}/dotenv-linter", 1)
```
Comment (SMillerDev) [inline Formula/dotenv-linter.rb:37]: I think it would be nicer to be a little more lenient here and do things like `assert_match "DuplicateKey", output` to make sure we don't need to update the test for whitespace changes upstream.

**Review by SMillerDev — APPROVED** [review]
Thanks @vladimyr  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉
